opennms-base-assembly/pom.xml

```diff
@@ -825,6 +825,38 @@
       <artifactId>org.opennms.features.newts</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <!-- config management -->
+    <dependency>
+      <groupId>org.opennms.features.config.dao</groupId>
+      <artifactId>org.opennms.features.config.dao.impl</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.opennms.features.config.dao</groupId>
+      <artifactId>org.opennms.features.config.dao.api</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.opennms.features.config.service</groupId>
+      <artifactId>org.opennms.features.config.service.impl</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.opennms.features.config.service</groupId>
+      <artifactId>org.opennms.features.config.service.api</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
```

Comment:
Some of these may not be required in the `lib/` folder if they are being loaded in OSGi.

Reply:
understand I will remove that for now and add it back when I work on RESTful api if needed.

Reply:
removed